cypress/fixtures/successful_university_login.json

```diff
@@ -3,6 +3,7 @@
     "id": 2,
     "email": "harvard@mail.com",
     "uid": "harvard@mail.com",
-    "role": "university"
+    "role": "university",
+    "subscriber": false
```

Comment:
Why do you need to mock the successful_university_login (here) and the response (above)?  If this is client side, don't you just need to mock the response?

Reply:
They are used in two different situations, the response is used when we're testing the actual University login, and for that we are actually writing the info on the UI. But the successful_university_login is used on the support/commands.js to have a valid University user we can use in other tests, like the Subscription button test that requires a Uni user to be logged in beforehand